### PR TITLE
nrf_wifi : Update Wi-Fi FW blobs

### PR DESCRIPTION
**Pull request by karun2796**
Pull the latest firmware fixes.

### CODE REVIEW

**Review by rajb9 — APPROVED** [review]

**Review by rajb9 — APPROVED** [review]

File: zephyr/module.yml [hunk 1]
```diff
@@ -1,42 +1,42 @@
 # Auto-generated file, do not edit, use update_wifi_fw.py to update
-name: nrf_wifi
+name: hal_nordic
```
Comment (krish2718) [inline zephyr/module.yml:2]: please rebase to fix this issue.

Comment (karun2796) [reply zephyr/module.yml:2]: Rebased. Do we need to run scripts/update_blobs.py again after rebasing?

Comment (krish2718) [reply zephyr/module.yml:2]: yes, please.

Comment (karun2796) [reply zephyr/module.yml]: Done

**Review by krish2718 — APPROVED** [review]

**Review by rajb9 — APPROVED** [review]

**Review by jukkar — APPROVED** [review]

**Review by rajb9 — APPROVED** [review]

**Review by rajb9 — APPROVED** [review]

**Review by rajb9 — APPROVED** [review]

**Review by rajb9 — APPROVED** [review]

**Review by rajb9 — APPROVED** [review]

**Review by rajb9 — APPROVED** [review]

**Review by rajb9 — APPROVED** [review]

**Comment by krish2718** [comment]
Merging based on previous twister state, as on the commit SHA has changed not the code (due to merge).

**Pull request merged**